test/unit/src/audio/FftUnit.cpp

```diff
@@ -3,7 +3,7 @@
 // FIXME: OOURA roundtrip FFT seems to be broken on windows for sizeFft = 4 (https://github.com/cinder/Cinder/issues/1263)
 #if ! defined( CINDER_MSW )
 
-#include "catch.hpp"
+#include "../catch.hpp"

```

Comment:
I'd imagine adding `INCLUDES "${UNIT_DIR}/src"` when calling `ci_make_app()` would do it. I can't really think of a more reasonable place for catch.hpp to live other than `cinder/test/unit/src/catch.hpp` but of course open to other places.